netbox/dcim/models/cables.py

```diff
@@ -219,6 +219,12 @@ def save(self, *args, **kwargs):
     def get_status_color(self):
         return LinkStatusChoices.colors.get(self.status)
 
+    def get_trace_url(self):
+        for cabletermination in self.terminations.all():
+            if cabletermination.termination_type.model in ['interface', 'rearport', 'frontport', 'consoleport', 'consoleserverport', 'powerport', 'poweroutlet']:
```

Comment:
This is generating a database query against ContentType for each termination. We'll have to devise a more efficient approach to resolving the trace URL.

Reply:
A littlebit higher in the code, there is a remark
```        # Query self.terminations.all() to leverage cached results```

So I assumed it will take the data from the cache in my call to